folium/map.py

```diff
@@ -297,6 +296,27 @@ def render(self, **kwargs):
             '</style>'), name='map_style')
 
         super(LegacyMap, self).render(**kwargs)
+    
+    @property
+    def location(self):
+        return self._location
```

Comment:
Not sure we need a class function. The location is used only once to create the map. We don't need to store it as a property b/c we won't do anything with it later. Just a private function to convert to float and issue a `ValueError` when that cannot be done would suffice IMO. Then you can run that function in `__init__()`, it would be just 1 extra line there. What do you think?